x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Meta } from '../../../../../common/types';
+
+export interface ApiLog {
+  timestamp: string; // Date ISO string
+  status: number;
+  http_method: string;
+  full_request_path: string;
+  user_agent: string;
+  request_body: string; // JSON string
+  response_body: string; // JSON string
+}
+
+export interface ApiLogsData {
+  results: ApiLog[];
+  meta: Meta;
```

Comment:
FYI, we get back some extra info in `meta`:

```
sort_direction: 'desc',
filters: {
  date: {
    from: // ... 
    to: // ...
  },
},
query: '',
```

But I'm not using it in the frontend (it's data we're sending to the server and not data we need to receive from the server), and I have no idea why `query` is even there, so I'm ignoring it